client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -10,6 +11,7 @@ import analyticsMixin from 'lib/mixins/analytics';
 import Notice from 'components/notice';
 import NoticeAction from 'components/notice/notice-action';
 import support from 'lib/url/support';
+import paths from 'my-sites/upgrades/paths';
```

Comment:
Good idea - I was blindly following the usual path of `import`ing path in its entirety, but just pulling in the one path used makes sense 👍